common/xrdp_client_info.h

```diff
@@ -143,11 +143,17 @@ struct xrdp_client_info
   int use_frame_acks;
   int max_unacknowledged_frame_count;
 
-  int disableSSLv3; /* 0 = no, 1 = yes */
+  int disableSSLv3; /* not in use anymore */
   char tls_ciphers[64];
 
   int client_os_major;
   int client_os_minor;
+
+  /* !!!!!!!!!!!!!!!!!!!!!!!!!! */
+  /* NO CHANGES ABOVE THIS LINE */
```

Comment:
If the goal is to keep xorgxrdp and X11rdp seeing the same fields they expect, the boundary should be after `options`.